org.gimp.GIMP.json

```diff
@@ -0,0 +1,76 @@
+{
+    "id": "org.gimp.GIMP",
+    "branch": "stable",
+    "base": "org.gimp.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "gimp-2.8",
+    "rename-desktop-file": "gimp.desktop",
+    "rename-appdata-file": "gimp.appdata.xml",
+    "rename-icon": "gimp",
+    "finish-args": ["--share=ipc", "--socket=x11", "--filesystem=host",
+                    "--filesystem=xdg-config/GIMP", "--share=network",
+                    "--socket=system-bus", "--socket=session-bus" ],
```

Comment:
Probably no reason to grant full bus access rather than just the few services needed.

Reply:
I'm not even sure what services uses GIMP, but probably quite a bit, and probably not all of them directly but through dependencies. I think that when I started writing this manifest (some time ago, so I forgot mostly), without this, a lot of things were broken.
I'm not sure how to proceed to list the needed services.

Reply:
`dbus-monitor` is a simple method (I think there are some gui tools too). You probably just use a few things like gvfs.

Reply:
Ok I replaced the full access by the gvfs only access. Hopefully I haven't missed anything.

Well at least "org.gnome.Shell.Screenshot" is missing, but it does not work, so for now I leave it out. Cf. https://github.com/flatpak/flatpak/issues/1090